Doc/whatsnew/3.13.rst

```diff
@@ -426,6 +426,18 @@ New Features
   APIs accepting the format codes always use ``Py_ssize_t`` for ``#`` formats.
   (Contributed by Inada Naoki in :gh:`104922`.)
 
+* The *keywords* parameter of :c:func:`PyArg_ParseTupleAndKeywords` and
+  :c:func:`PyArg_VaParseTupleAndKeywords` has now type :c:expr:`char * const *`
+  in C and :c:expr:`const char * const *` in C++, instead of :c:expr:`char **`.
+  It makes these functions compatible with argument of type
+  :c:expr:`const char * const *`, :c:expr:`const char **` or
+  :c:expr:`char * const *` in C++ and :c:expr:`char * const *` in C
+  without explicit type cast.
+  This can be overridden by defining the macro ``PY_CXX_CONST``
+  before including :file:`Python.h` as ``const`` for the latter and as
+  empty value for the former.
```

Comment:
```suggestion
  This can be overridden with the :c:macro:`PY_CXX_CONST` macro.
```

Now that this is documented, simply refer interested readers here directly instead of duplicating the whole explanation.